src/transport.js

```diff
@@ -1,26 +1,99 @@
-import request from 'request-promise'
-import promiseRetry from 'promise-retry'
+import fetch from 'cross-fetch'
+// from https://github.com/jonbern/fetch-retry
+// and https://github.com/jonbern/fetch-retry/pull/27
+export default (url, options) => {
+  var retries = 3
+  var retryDelay = 1000
+  var retryOn = [429, 502, 503, 504]
+  if (options && options.retries !== undefined) {
+    if (isPositiveInteger(options.retries)) {
```

Comment:
can't you just check `options.retries > 0`?

Reply:
Maybe we could, but I let it that way, the original also checks for Integer